groups/infrastructure/profiles/staging-eu-west-2/stagsbox/vars

```diff
@@ -0,0 +1,5 @@
+environment = "stagsbox"
+aws_profile = "staging-eu-west-2"
+enable_container_insights = true
+
+ec2_key_pair_name = "ch-aws-staging"
```

Comment:
Looks good, worth adding in live ahead of time?

Reply:
As long it's not triggered... ;-)